experimental/trusted_shuffler/server/src/http.rs

```diff
@@ -88,33 +131,63 @@ fn trusted_shuffler_to_hyper_request(
     trusted_shuffler_request: TrustedShufflerRequest,
 ) -> Request<Body> {
     let body = Body::from(trusted_shuffler_request.body);
-    Request::builder()
+
+    let mut request = Request::builder()
         .uri(trusted_shuffler_request.uri)
         .method(Method::POST)
         .version(http::Version::HTTP_2)
         .body(body)
-        .expect("Failed to convert Trusted to Hyper Request")
+        .expect("Failed to convert Trusted to Hyper Request");
```

Comment:
`expect` crashes the whole program if it fails
I think it's better to use `.context()` and return `anyhow::Result`